### ISSUE TITLE
wishlist: Please allow (some) registry settings to be in HKLM

### ISSUE THREAD

**Issue by feltel** [aluxnimm/outlookcaldavsynchronizer#301]
We are using and distributing Caldav Syncronizier within our network with the help of OPSI, a software installation system. In the past we did some pre-sets (i.e. disabling update check, language, ...) with the help of some registry settings and populated to the machines with GPO-Group Policy Preferences. Over the last month we have a growing number of machines which are not part of our domain network an so do not have access to GPO settings. Caldav syncronizer stores its settings in the HKEY_CURRENT_USER tree of the registry, so every user got its own settings tree upon using the software. It would be awesome if at least some of those registry settings (to disable the update check) would be honored in HKEY_CURRENT_MACHINE and have precedence of those in HKCU. If this would be implemented we could distribute those settings upon software installation, even to machines which are not domain joined or even to disconnected machines.

**Comment by aluxnimm** [comment]
Good point, will be considered. Meanwhile, did you check the app.config file? Some of the general options can be configured there as well and those values are used if no HKCU key is set yet. And you should be able to distribute that file via OPSI as well during installation.
